packages/components/src/internal/components/search/FilterExpressionView.tsx

```diff
@@ -324,6 +324,8 @@ export const FilterExpressionView: FC<Props> = memo(props => {
                 const expanded = expandedOntologyKey === ontologyBrowserKey;
                 // FIXME: This is not a proper usage of Dropdown, as it is not rendering any MenuItems this behavior
                 //  should be accomplished some other way (OverlayTrigger + Popover with click events?)
+                //  Note: we can just replace this with some basic logic to show/hide it and render it inline, it does
+                //  not need to render as a floating item, so Dropdown is very unnecessary.
```

Comment:
I assume this is for another / separate PR?

Reply:
Yep, going to handle this case with the ProductMenu et. al.